### ISSUE TITLE
[BITV]: Create possibility for `NcActions` to be automatically closed (removed from DOM) if they are not visible anymore

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42702]
There are several places in our instance where `NcActions` overlapping another content and flying out of the content where they belongs to. Examples:

![Peek 2024-01-11 11-05](https://github.com/nextcloud/server/assets/6078378/ddf47c7a-5615-4401-ab3a-faa95aec3572)

![Peek 2024-01-11 09-50](https://github.com/nextcloud/server/assets/6078378/13303849-fd08-46d4-9146-6525f8dfbe8a)

![Peek 2024-01-11 09-49](https://github.com/nextcloud/server/assets/6078378/4ea02616-3d99-4c25-a522-509fcd31b770)

Please create a possibility to remove `NcActions` from DOM as far the actions button is not visible anymore (user scrolled up/down a content). @ShGKme suggested to use `IntersectionObserver` fro this purposes.

**Comment by JuliaKirschenheuter** [comment]
It is ok to remove `NcActions` from the DOM because only following requirements to be noted: https://bitvtest.de/bitv_test/das_testverfahren_im_detail/vertiefend/ueberarbeitung/2019.html, under "1.4.13a Eingeblendete Inhalte bedienbar"
